.github/workflows/openstack/terraform/run_elevate.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -x
+
+wget -O /scripts/elevate-cpanel https://raw.githubusercontent.com/cpanel/elevate/release/elevate-cpanel
+chmod -v 700 /scripts/elevate-cpanel
+/scripts/elevate-cpanel --check
+/usr/local/cpanel/bin/whmapi1 start_background_mysql_upgrade version=10.6
```

Comment:
FIX: This is not needed and should be removed so that we test that MySQL gets updated as expected